### PR TITLE
adding solution and test files

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: test/primeToX.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  const expected =solution(num);
+  compareArr(expected,result);
+  if(compareArr(expected,result)==true){
+    console.log("Both arrays are the same");
+  }
+  else{
+    console.log("Arrays are incorrect");
+  }
+}
+test(7,[7,5,3,2]);
```
Comment (allopez7) [inline test/primeToX.js:11]: test it about two more times.

**Review by allopez7 — APPROVED** [review]

File: solutions/primeToX.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const isPrime=(num,i=2)=>{
```
Comment (kinuesantos) [inline solutions/primeToX.js]: require your previous function, instead of re-writing it.

File: solutions/primeToX.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    return false;
+  }
+  return isPrime(num,i+1)
+}
+
+const iterate=(num,arr=[])=>{
+  if(num==1){
+    return arr;
+  }
+  if (isPrime(num)==true){
```
Comment (kinuesantos) [inline solutions/primeToX.js]: No need for ==true.

File: test/primeToX.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+const solution = require('../solutions/primeToX.js');
+
+const compareArr=(a1,a2,i=0)=>{
```
Comment (kinuesantos) [inline test/primeToX.js]: No need for this function. Just compare your solution to the result

File: test/primeToX.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  if(i==a1.length){
```
Comment (roycep) [inline test/primeToX.js]: Be careful of this statement.

If your algorithm prints out [1,2,3] (a1) when the real answer is [1,2,3,4] (a2),
Your algorithm will reach i = 3 and return true.
But the thing is that [1,2,3] and [1,2,3,4] are not equal.

**Review by kinuesantos — APPROVED** [review]

**Pull request merged**